messages/en.js

```diff
@@ -13,35 +13,35 @@ Blockly.Msg.NODE_OBJECT_SET_JSON = "set %1 property %2 to %3";
 Blockly.Msg.NODE_OBJECT_SET_JSON_TOOLTIP = "Set a property of an object.";
 Blockly.Msg.NODE_OBJECT_SET_WARNING = "Nested properties can only be set with string literals";
 Blockly.Msg.NODE_LOG = "log text %1 with %2 level";
-Blockly.Msg.NODE_LOG_TOOLTIP = "Log via the Node-Red function API.";
-Blockly.Msg.NODE_LOG_LOG = "log"; 
+Blockly.Msg.NODE_LOG_TOOLTIP = "Log via the Node-RED function API.";
+Blockly.Msg.NODE_LOG_LOG = "log";
 Blockly.Msg.NODE_LOG_WARNING = "warning";
 Blockly.Msg.NODE_LOG_ERROR = "error";
 Blockly.Msg.NODE_LOG_DEBUG = "debug";
 Blockly.Msg.NODE_LOG_TRACE = "trace";
 Blockly.Msg.NODE_MSG = "msg";
-Blockly.Msg.NODE_MSG_TOOLTIP = "Get the Node-Red input message.";
+Blockly.Msg.NODE_MSG_TOOLTIP = "Get the Node-RED input message.";
 Blockly.Msg.NODE_REMOVE_STATUS = "remove node status";
-Blockly.Msg.NODE_REMOVE_STATUS_TOOLTIP = "Remove the node status via the Node-Red function API.";
+Blockly.Msg.NODE_REMOVE_STATUS_TOOLTIP = "Remove the node status via the Node-RED function API.";
 Blockly.Msg.NODE_CLONE = "clone %1";
-Blockly.Msg.NODE_CLONE_TOOLTIP = "Clone the message via the Node-Red function API.";
+Blockly.Msg.NODE_CLONE_TOOLTIP = "Clone the message via the Node-RED function API.";
 Blockly.Msg.NODE_STATUS = "set node status to text %1 to color %2 and shape %3";
-Blockly.Msg.NODE_STATUS_TOOLTIP = "Set the node status via the Node-Red function API.";
-Blockly.Msg.NODE_STATUS_RING = "ring"; 
+Blockly.Msg.NODE_STATUS_TOOLTIP = "Set the node status via the Node-RED function API.";
+Blockly.Msg.NODE_STATUS_RING = "ring";
 Blockly.Msg.NODE_STATUS_DOT = "dot";
 Blockly.Msg.NODE_PROPERTIES = "get node %1";
-Blockly.Msg.NODE_PROPERTIES_TOOLTIP = "Get the specified node property via the Node-Red function API.";
+Blockly.Msg.NODE_PROPERTIES_TOOLTIP = "Get the specified node property via the Node-RED function API.";
 Blockly.Msg.NODE_PROPERTIES_ID = "identifier";
 Blockly.Msg.NODE_PROPERTIES_NAME = "name";
 Blockly.Msg.NODE_PROPERTIES_OUTPUT_COUNT = "output count";
 Blockly.Msg.NODE_FLOW_MEMORY = "flow";
-Blockly.Msg.NODE_FLOW_MEMORY_TOOLTIP = "Access the data in the Node-Red flow memory.";
+Blockly.Msg.NODE_FLOW_MEMORY_TOOLTIP = "Access the data in the Node-RED flow memory.";
 Blockly.Msg.NODE_CONTEXT_MEMORY = "(node)context";
```

Comment:
What is `(node)context` exactly? Should there be a space? Translations carry on this term literally.

Reply:
The (node)context is simply the context memory of a node.  As you can see [here](https://nodered.org/docs/user-guide/context) you have node context, flow context, and global context.  In case of flow and global, the 'context' is not mentioned in our blockly toolbar.  After a discussion on Discourse at the time being, it was decided that to to it like that.  Personally I found it more readable if the 'context' was mentioned in all 3 blocks...

Reply:
Node-RED isn't the clearest when it comes to naming things - worst culprit is the over used term "flow" :)
Node-RED just calls the node local storage object "context" whereas it calls the others "global context" and "flow context" so I suggested using "(node) context" to try to make things clearer but using the ( ) to show that it wasn't official Node-RED name.

Reply:
Ok, so the missing space was a typo, so I used this assumption to edit.